### PR TITLE
Update security-defaults.md

### PR DESCRIPTION
**Pull request by Sammak**
Note for MFA skip removal

### CODE REVIEW

**Comment by Court72** [comment]
@MicrosoftGuyJFlo

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Sammak** [comment]
@MicrosoftGuyJFlo Did you get a chance to review this?

**Review by MicrosoftGuyJFlo — APPROVED** [review]

**Comment by MicrosoftGuyJFlo** [comment]
#resolves https://github.com/MicrosoftDocs/entra-docs/pull/851

**Comment by MicrosoftGuyJFlo** [comment]
#sign-off

File: docs/fundamentals/security-defaults.md [hunk 1]
```diff
@@ -70,6 +70,9 @@ As part of enabling security defaults, administrators should revoke all existing
 
 ### Require all users to register for Microsoft Entra multifactor authentication
 
+> [!NOTE]
+> Starting July 29th, 2024 new tenants may not have the 14 day grace period for users to register for MFA. We are making this change to help reduce the risk of account compromise during the 14 day window, as MFA can block over 99.2% of identity-based attacks.
```
Comment (Jak-MS) [inline docs/fundamentals/security-defaults.md:74]:
```suggestion
> Starting July 29th, 2024 new tenants may not have the 14-day grace period for users to register for MFA. We are making this change to help reduce the risk of account compromise during the 14-day window, as MFA can block over 99.2% of identity-based attacks. 
```

**Pull request merged**